apps/aecore/src/aec_mining.erl

```diff
@@ -45,7 +44,7 @@ mine(Attempts) ->
 -spec mine(block(), non_neg_integer()) -> {ok, block()} | {error, term()}.
 mine(Block, Attempts) ->
     Target = aec_blocks:target(Block),
-    {ok, BlockBin} = aec_headers:serialize_for_network(aec_blocks:to_header(Block)),
+    {ok, BlockBin} = aec_headers:serialize_to_binary(aec_blocks:to_header(Block)),
```

Comment:
Should this not use `aec_headers:serialize_to_network/1` instead?

Reply:
I expect `serialize_to_binary` is meant to be canonical serialization. Comment expressing this would help.